lib/device-log/ios-log.js

```diff
@@ -199,16 +203,30 @@ class IOSLog {
   }
 
   async stopCapture () {
-    logger.debug('Stopping iOS log capture');
-    if (this.proc && this.proc.isRunning) {
+    if (this.proc) {
+      if (this.udid) {
+        // If no other UDID's are using 'idevicesyslog' kill it
+        if (--IOSLog.cachedIDeviceSysLogs[this.subprocessId].count === 0) {
```

Comment:
What is this post-decrement doing? Seems like a side-effect waiting to cause a bug. If this `count` needs to be decremented, I'd rather it happen explicitly, rather than in the conditional.